### ISSUE TITLE
`StoreImageSequence` leaks GPU memory

### ISSUE THREAD

**Issue by PathogenDavid** [bonsai-rx/bonsai#1775]
Found while investigating https://github.com/bonsai-rx/bonsai/issues/1645

`StoreImageSequence` will leaks GPU memory when processed continuously.

## Repro

1. Download [TextureLeakIssue.zip](https://github.com/bonsai-rx/bonsai/files/15204537/TextureLeakIssue.zip) and open the enclosed workflow
2. Open Task Manager and navigate to the GPU section under the Performance tab
3. Start the workflow
4. Observe your GPU memory usage shoot to the moon (note: it is expected for the rendered camera to be laggy, that's not a symptom it's just what the workflow is meant to do)

## Cause and proposed fix

I did not investigate completely, but I believe this happens because `StoreImageSequence` never disposes of the `TextureSequence`, nor does it re-use and update the same one continuously as one might expect.

This effectively makes it the opposite of https://github.com/bonsai-rx/bonsai/issues/1774

I believe the proper fix will be the same as that one, and once again will benefit from the proposed solution to https://github.com/bonsai-rx/bonsai/issues/1773

**Comment by glopesdev** [comment]
@PathogenDavid I agree the current semantics of this operator is confusing. The intention here was that textures created by `StoreImageSequence` were to be explicitly disposed, as described on my comment in #1774. However, it seems like this operator ended up being co-opted for a few different purposes which are not consistent with `StoreImage`.

This operator is seldom used to be honest, and I suspect it's design was just not as thoroughly tested as the other operators, so we can review it for release 2.9.
